### PR TITLE
Fix when a single scene number is entered it was saved as 1x(the numb…

### PR DESCRIPTION
**Pull request by p0psicles**
This was only an issue for multi season shows.
Single season shows already had proper sanity checks. For example a single season show like One-Punch man. Entering an alphanumeric char, resets to the absolute episode number.

Same applies for multi season shows. But there you can use the format: [season]x[episode].
This fix makes sure entering an alphanummeric char resets to 0x0. And entering a single digit for season 12 will result in 12x[single digit].

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
Bwaha, commented on the other one before I saw this

**Comment by miigotu** [comment]
@p0psicles only thing I need to know is, what if the episode is 2x1 and you need to change it to 1x20, still works?

**Comment by miigotu** [comment]
Nevermind, checked and looks like that is handled.

File: gui/slick/js/core.js [hunk 1]
```diff
@@ -2324,23 +2324,42 @@ var SICKRAGE = {
                 });
             }
 
+            function setInputValidInvalid(valid, el) {
+            	if (valid) {
+            		$(el).css({'background-color': '#90EE90','color': '#FFF', 'font-weight': 'bold'}); //green
+            		return true;
+            	}
+            	else {
```
Comment (OmgImAlexis) [inline gui/slick/js/core.js:2332]: Can js please stop being merged until @p0psicles uses the correct formatting. This should be on a single line. This isn't Python or any other language.

`} else {` is how the WHOLE rest of the file is done.

Comment (p0psicles) [inline gui/slick/js/core.js:2332]: Keep your pants on. Ill put A new PR out for the proper formatting. And nobody holding you back to do the same.

Comment (OmgImAlexis) [inline gui/slick/js/core.js:2332]: I shouldn't have to fix your stuff though. I keep telling you about this.

Comment (miigotu) [inline gui/slick/js/core.js:2332]: Play nice...

@OmgImAlexis he is still getting his feet wet, he will find his groove.
@p0psicles just follow the convention in whatever file you are in.

**Pull request merged**